src/CommunityToolkit.Maui.Core/Essentials/FileSaver/FileSaverImplementation.android.cs

```diff
@@ -14,10 +14,13 @@ public sealed partial class FileSaverImplementation : IFileSaver
 {
 	static async Task<string> InternalSaveAsync(string initialPath, string fileName, Stream stream, CancellationToken cancellationToken)
 	{
-		var status = await Permissions.RequestAsync<Permissions.StorageWrite>().WaitAsync(cancellationToken).ConfigureAwait(false);
-		if (status is not PermissionStatus.Granted)
+		if (Android.OS.Build.VERSION.SdkInt < Android.OS.BuildVersionCodes.Tiramisu)
```

Comment:
it should be fixed in .NET MAUI. please revert this change

Reply:
The example app does not work on Android 13 without this, I tried yesterday, all latest versions.

Reply:
If the fix will leave here, please use this API for check the android version instead

```suggestion
                    if (OperatingSystem.IsAndroidVersionAtLeast(33))
```

Reply:
Thanks, didn't know about that, it seems more intuitive.